GitHubContents.ps1

```diff
@@ -311,6 +341,10 @@ function Set-GitHubContent
             ParameterSetName = 'Elements')]
         [string] $RepositoryName,
```

Comment:
I prefer for the parameters to be in the same order as their CBH entries.  `OwnerName`/`RepositoryName` should really be at the top before `Uri`.  I'm ok with the fact that they'd be listed before a parameter with `Position = 1`.